data-transformers/transformations/dataTransformations.py

```diff
@@ -1,3 +1,3 @@
 class Transformations():
-    def secondsToMilliseconds(s):
+    def secondsToMilliseconds(self, s):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/BenchFlow/data-transformers/file/2615772760/issues/source?bid=3301038&fileBranchId=3301041#l2)